### PR TITLE
Created a rough draft of Profile Page

### PR DESCRIPTION
**Pull request by nkonjeti**

### CODE REVIEW

**Comment by coconut750750** [comment]
Looks pretty good

File: frontend/src/pages/Profile.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  const [longDesc, setLongDesc] = useState();
+  const [numRoomates, setNumRoomates] = useState();
+  const [fbUsername, setfbUsername] = useState();
+  const [igUsername, setigUsername] = useState();
+  const [scUsername, setscUsername] = useState();
+ //callApi().then(text=> setText(text));
+ 
+ useEffect(() => {
+  const getText = async () => {
+  const data = await loadProfile("5fadf6bba3e3324e794a1141");
```
Comment (alicesf2) [inline frontend/src/pages/Profile.js]: I would suggest changing the variable name to something more specific, like profile

File: frontend/src/styles/Profile.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
+.profile-mediaIcons {
```
Comment (alicesf2) [inline frontend/src/styles/Profile.css:1]: good use of css classes :D

File: backend/index.js [hunk 3]
```diff
@@ -18,10 +18,20 @@ const Profile = require("./models/profile.js");
...
-app.get('/', async (req, res) => {
-  res.send('Hello World')
+
+//Bind connection to error event (to get notification of connection errors)
+db.on('error', console.error.bind(console, 'MongoDB connection error:'));
+
+//const Pusheens = require("./models/Pusheens.js");
+
+
+app.get('/profilepage/:userid', async (req, res) => {
```
Comment (alicesf2) [inline backend/index.js]: nitpick: try to camelcase all javascript variables (i.e. userid -> userId)

File: frontend/src/utils/api.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export async function loadProfile(userid) {
```
Comment (alicesf2) [inline frontend/src/utils/api.js]: change this to userId too

**Review by alicesf2 — COMMENTED** [review]
Looks good :nehalapeno:! I'll wait until you finish the edit profile page to approve

File: backend/index.js [hunk 5]
```diff
@@ -40,11 +41,65 @@ app.use("/auth", authRoute);
 app.use("/profile" , profileRoute);
 app.use("/register", registerRoute)
 
+
+//Bind connection to error event (to get notification of connection errors)
+
+//const Pusheens = require("./models/Pusheens.js");
```
Comment (coconut750750) [inline backend/index.js]: get rid of some of these comments

**Review by coconut750750 — CHANGES_REQUESTED** [review]

File: backend/index.js (hunk 5 above)
Comment (nkonjeti) [reply backend/index.js]: 👍🏽

**Review by coconut750750 — APPROVED** [review]

**Pull request merged**